lib/vsc/administration/slurm/sync.py

```diff
@@ -317,15 +317,22 @@ def slurm_user_accounts(vo_members, active_accounts, slurm_user_info, clusters,
         for m in members:
             reverse_vo_mapping[m] = (vo.vsc_id, vo.institute['name'])
 
+    remove_users = set()  # we do not make a disctinction between clusters
+
     for cluster in clusters:
         cluster_users_acct = [
             (user.User, user.Def_Acct) for user in slurm_user_info if user and user.Cluster == cluster
         ]
         cluster_users = set([u[0] for u in cluster_users_acct])
 
-        # these are the users that need to be removed as they are no longer an active user in any
+        # these are the users that need to be completely removed as they are no longer an active user in any
```

Comment:
where does the comment apply to? `remove_users_cluster` or `remove_users`

Reply:
Both.